src/metrics/loc.rs

```diff
@@ -377,20 +377,21 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_blank_python() {
+    fn test_blank() {
         check_metrics!(
-            "\na = 42\n\n",
+            "
+            a = 42
+            \n",
             "foo.py",
             PythonParser,
             loc,
             [(blank, 1, usize)]
         );
-    }
 
-    #[test]
-    fn test_blank_rust() {
```

Comment:
Maybe better to keep the tests as separated, so they can pass/fail individually

Reply:
It's better to group tests on the same stuff in a single function, at least we immediately know the wrong metric. Having the same metric test for each language is a boilerplate to maintain and eventually update

Reply:
If we keep them separated, we still immediately know the wrong metric, but we have the possibility to know if a specific language is broken.

Reply:
I can't figure out the problem, sorry :)
Since each sub-test of a metric test is a macro, if a metric test is failing, we can immediately detect which macro is involved and at the same time we can determine if that specific language is broken.

Reply:
If there is a failure on the first assertion, the other assertions won't be evaluated.

In this case, if parsing Python fails, we won't parse Rust and C too, so we won't be able to tell right away if it's just a Python problem or if is a cross-language problem.

Reply:
> If there is a failure on the first assertion, the other assertions won't be evaluated.
> 
> In this case, if parsing Python fails, we won't parse Rust and C too, so we won't be able to tell right away if it's just a Python problem or if is a cross-language problem.

Oh, so it's just a comfort matter. I thought to solve that problem commenting the series of sub-test macros until the botched one is found. Because if a test is failing we should get the hands on the same to discover the cause.

Actually I refactored the code because I didn't like all those little tests for the same thing

Reply:
Anyway, I see your point, but I think it would be clear to have a test-per-metric instead of different test-per-metric-per-language as I thought initially

Reply:
Well, since there is no practical advantage in sticking the tests together, but there is an advantage in keeping them separated, let's keep them separated.

Reply:
I don't like this approach, but if it is more comfortable for you, let's keep them as I thought initially :P